ui/helpers/job.js

```diff
@@ -180,28 +180,33 @@ export const getResultState = function getResultState(job) {
 export const addAggregateFields = function addAggregateFields(job) {
   const {
     job_group_name: jobGroupName,
-    job_group_symbol: jobGroupSymbol,
     job_type_name: jobTypeName,
-    job_type_symbol: jobTypeSymbol,
     platform,
     platform_option: platformOption,
     submit_timestamp: submitTimestamp,
     start_timestamp: startTimestamp,
     end_timestamp: endTimestamp,
   } = job;
+  let { job_type_symbol: jobTypeSymbol } = job;
 
   job.resultStatus = getResultState(job);
-  // we want to join the group and type information together
-  // so we can search for it as one token (useful when
-  // we want to do a search on something like `fxup-esr(`)
-  const symbolInfo = jobGroupSymbol === '?' ? '' : jobGroupSymbol;
 
+  // The current modification is to support backfilling of manifest based scheduling.
+  // A backfilled task (e.g. bc2) ends up being named bc2-<revision>-bk
+  // The label can also be different than the original task selected to be backfilled
+  // For instance 'test-linux1804-64/debug-mochitest-browser-chrome-e10s-4' can be
+  // 'test-linux1804-64/debug-mochitest-browser-chrome-e10s-1' yet the symbol be bc4
+  const chunk = Number(jobTypeName.split('-').pop());
+  if (Number.isInteger(chunk)) {
+    jobTypeName.replace(`-${chunk}`, '');
```

Comment:
Doing a `replace` here works, but carries some risk.  What if there was a name like `foo-1bar-baz-1`?  It would remove the `-1` from the `-1bar`.

You might consider doing something like:

    const parts = Number(jobTypeName.split('-');
    const chunk = parts.pop());
    const joinedParts = parts.join('-');

    jobTypeName = Number.isInteger(chunk) ? joinedParts : `${joinedParts}-${chunk}`;

Or, I guess an `if...then` above to handle the `jobTypeSymbol` change below, too.  But you get the idea.  :)